### ISSUE TITLE
The highlighted color in the red box cannot be displayed.

### ISSUE THREAD

**Issue by xxxdo1** [winsiderss/systeminformer#2286]
### Brief description of your issue

The highlighted color in the red box cannot be displayed, it only shows the two colors above.

![2024111819032784](https://github.com/user-attachments/assets/60ba8585-71bd-4af7-a178-9c494838a2e2)

### Environment (optional)

```shell
Version 3.1.24283.0
```

**Comment by dmex** [comment]
> The highlighted color in the red box cannot be displayed, it only shows the two colors above.

The area you've highlighted red shows clear white boxes just before each color. Removing those checkmarks is how you disable highlighting for those colors.

![image](https://github.com/user-attachments/assets/d783870d-723c-4288-ab71-602e2a39ff5b)

You can re-enable highlighting by clicking those boxes to re-enable the checkmarks. When you close the settings window the highlighting will be enabled again for each window or you click the `Enable All` button to re-enable all highlighting defaults: 

![image](https://github.com/user-attachments/assets/8bcd00c0-ff31-4f42-8291-a21d4bfa333b)

**Comment by xxxdo1** [comment]
I know, I checked the three checkboxes above, and the program running with high-privilege administrator is not highlighted

The following programs are run with administrator privileges in scheduled tasks.
![20241119689967](https://github.com/user-attachments/assets/7634a506-e6fb-4b75-bb7c-da3d2c88a0a3)

**Comment by dmex** [comment]
> The following programs are run with administrator privileges in scheduled tasks.

Is the token for those processes the current user account or did task scheduler execute them with a different user?

**Comment by xxxdo1** [comment]
Yes, I am using an administrator account, and all programs in the task scheduler are run using this account.

![20241119173540112](https://github.com/user-attachments/assets/19c90e91-03ec-438d-b2d9-5ba03b4e9f75)
